service/storage_service.cc

```diff
@@ -1139,6 +1139,11 @@ class topology_coordinator {
                     "topology coordinator attempted to enable features even though there are joining nodes");
         }
 
+        if (utils::get_local_injector().enter("raft_topology_suppress_enabling_features")) {
+            co_await sleep(std::chrono::milliseconds(100));
```

Comment:
Why the sleep though?

Reply:
While this error injection is enabled and the coordinator is ready to enable features, it will keep entering this path and the sleep is a way to throttle the retries.

Reply:
Comment please

Reply:
Added